.mbedignore

```diff
@@ -4,3 +4,4 @@ fuzzing/
 scripts/
 test/
 third-party/
+extras/
```

Comment:
I think this is a good opportunity to remove old folders from the list (fuzzing, scripts, test, and third-party).
Can you do that?

Reply:
I have added this in dc33e42